backend/src/types/DataTypes.d.ts

```diff
@@ -37,7 +37,7 @@ export type DBTeamEventAttendance = {
   hoursAttended?: number;
   image: string;
   eventUuid: string;
-  pending: boolean;
+  pending?: boolean;
```

Comment:
I think we'll be ok to remove this field entirely, since we're not using it. Note that even if the data in the database has this field, you won't have a "type error" due to extra fields that the type doesn't have.